front/src/TurtleEnableBar.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import TurtleEnableButton from "./TurtleEnableButton";
+import styled from "styled-components";
+
+/**
+ * Simple bar with all the turtles
+ * Author: S.A. Tanja
+ * Author: H.E. van der Laan
+ */
+
+const TurtleEnableBar = props => {
+  return (
+    <div className={props.className}>
+      {props.turtles.map((turtle, position) => {
+        return (
+          <TurtleEnableButton
+            key={turtle.id}
+            enabled={turtle.enabled}
+            id={position + 1}
+            onEnable={() => props.onEnable(position)}
+            onDisable={() => props.onDisable(position)}
+          />
+        );
+      })}
+    </div>
+  );
+};
+
+const TurtleBar = styled(TurtleEnableBar)`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  overflow-x: auto;
+  align-content: space-between;
+  border-style: solid;
+  border-width: 0px 0px 2px 0px;
```

Comment:
In the coding guidlines it says we shouldn't be using `px`. Instead, use `rem`. 1 `rem` = 10 `px` (see `index.css`).

Reply:
aight